kodev

```diff
@@ -1128,11 +1128,19 @@ TARGET:
 
     case "${1}" in
         android)
-            if [ -n "${KODEBUG}" ]; then
-                adb logcat 'KOReader:* ActivityManager:* AndroidRuntime:* DEBUG:* *:F dlopen:* LuaJIT:*'
-            else
-                adb logcat 'KOReader:I ActivityManager:* AndroidRuntime:* DEBUG:* *:F'
-            fi
+            command -v pidcat >/dev/null && {
```

Comment:
You don't think a regular if/else would be a touch more obvious at a glance?

Reply:
Actually scratch that:

```
In kodev line 1131:
            command -v pidcat >/dev/null && {
                                         ^-- SC2015: Note that A && B || C is not if-then-else. C may run when A is true.
```

Of course, what it says makes sense. It'll run C if B is false. It doesn't feel like that the intent?